anime_downloader/sites/kissanimex.py

```diff
@@ -0,0 +1,41 @@
+from anime_downloader.sites import helpers
+from anime_downloader.sites.anime import Anime, AnimeEpisode, SearchResult
+
+class KissAnimeX(Anime, sitename = 'kissanimex'):
+    sitename = "kissanimex"
+
+    @classmethod
+    def search(cls, query):
+        url = 'https://kissanimex.com/search?url=search'
+        r = helpers.get(url, params={'q': query})
+        soup = helpers.soupify(r.text)
+        items = soup.find_all('td')
+        search_results = [
+                SearchResult(
+                    title = x.find('a').text,
+                    url = 'https://kissanimex.com' + x.find('a')['href']
+                    )
+                for x in items
+                ]
+        return search_results
+
+    def _scrape_episodes(self):
+        r = helpers.get(self.url).text
+        eps = helpers.soupify(r).find('div', id='episodes-sub').select('td > a')
```

Comment:
```suggestion
        eps = helpers.soupify(r).find('div#episodes-sub > td > a')
```
probably works

Reply:
it doesnt work for me

Reply:
Maybe it's because he assumed it's a direct child? Try: `eps = helpers.soupify(r).find('div#episodes-sub td > a')`